src/test/TestGhoBucketCapacitySteward.t.sol

```diff
@@ -0,0 +1,212 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import './TestGhoBase.t.sol';
+
+contract TestGhoBucketCapacitySteward is TestGhoBase {
+  address internal NEW_OWNER = makeAddr('NEW_OWNER');
+
+  function setUp() public {
+    // Deploy Gho Bucket Capacity Steward
+    GHO_BUCKET_CAPACITY_STEWARD = new GhoBucketCapacitySteward(
+      SHORT_EXECUTOR,
+      address(GHO_TOKEN),
+      RISK_COUNCIL
+    );
+    address[] memory controlledFacilitators = new address[](2);
+    controlledFacilitators[0] = address(GHO_ATOKEN);
+    controlledFacilitators[1] = address(GHO_GSM);
+    vm.prank(SHORT_EXECUTOR);
+    GHO_BUCKET_CAPACITY_STEWARD.setControlledFacilitator(controlledFacilitators, true);
+
+    /// @dev Since block.timestamp starts at 0 this is a necessary condition (block.timestamp > `MINIMUM_DELAY`) for the timelocked contract methods to work.
+    vm.warp(GHO_BUCKET_CAPACITY_STEWARD.MINIMUM_DELAY() + 1);
+
+    // Grant roles
+    GHO_TOKEN.grantRole(GHO_TOKEN_BUCKET_MANAGER_ROLE, address(GHO_BUCKET_CAPACITY_STEWARD));
+  }
+
+  function testConstructor() public {
+    assertEq(GHO_BUCKET_CAPACITY_STEWARD.owner(), SHORT_EXECUTOR);
+    assertEq(GHO_BUCKET_CAPACITY_STEWARD.GHO_TOKEN(), address(GHO_TOKEN));
+    assertEq(GHO_BUCKET_CAPACITY_STEWARD.RISK_COUNCIL(), RISK_COUNCIL);
+
+    address[] memory controlledFacilitators = GHO_BUCKET_CAPACITY_STEWARD
+      .getControlledFacilitators();
+    assertEq(controlledFacilitators.length, 2);
+
+    uint40 facilitatorTimelock = GHO_BUCKET_CAPACITY_STEWARD.getFacilitatorBucketCapacityTimelock(
+      controlledFacilitators[0]
+    );
+    assertEq(facilitatorTimelock, 0);
+  }
+
+  function testRevertConstructorInvalidExecutor() public {
+    vm.expectRevert('INVALID_OWNER');
+    new GhoBucketCapacitySteward(address(0), address(0x002), address(0x003));
+  }
+
+  function testRevertConstructorInvalidGhoToken() public {
+    vm.expectRevert('INVALID_GHO_TOKEN');
+    new GhoBucketCapacitySteward(address(0x001), address(0), address(0x003));
+  }
+
+  function testRevertConstructorInvalidRiskCouncil() public {
+    vm.expectRevert('INVALID_RISK_COUNCIL');
+    new GhoBucketCapacitySteward(address(0x001), address(0x002), address(0));
+  }
+
+  function testChangeOwnership() public {
```

Comment:
add the revert case pls. Also, I think NEW_OWNER can be moved to this function (no need for a contract constant)

Reply:
Done

Reply:
macro case is only for constant/immutables. Let's use `newOwner` instead

Reply:
Done